app/views/contributions/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<section class="section">
+  <h2>Request for Contribution</h2>
+  <h1 class="title"><%= contribution.type %>: <%= contribution.all_tags_unique.map(&:humanize).join(", ") %></h1>
```

Comment:
This is nitpicky, but i'm not sure its good practice to put an `h2` before an `h1`?

Reply:
You're absolutely right. I took care of that.